readthedocs_ext/external_version_warning.py

```diff
@@ -16,6 +17,9 @@ def process_external_version_warning_banner(app, doctree, fromdocname):
     If the version type is external this will show a warning banner
     at the top of each page of the documentation.
     """
+    if not isinstance(doctree, addnodes.document):
+        return
```

Comment:
I'd like to see a comment here explaining what you've explained in the PR in simpler words.

Reply:
I think [67c43c1155a48294a9abaa9c943f923f8bde55f7](https://github.com/readthedocs/readthedocs-sphinx-ext/pull/114/commits/67c43c1155a48294a9abaa9c943f923f8bde55f7) is enough